jwst/tests_nightly/general/nirspec/test_detector1.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_detector1 import Detector1Pipeline
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_detector1pipeline4(_bigdata):
+    """
+
+    Regression test of calwebb_detector1 pipeline performed on NIRSpec data.
+
+    """
+    step = Detector1Pipeline()
+    step.save_calibrated_ramp = True
+    step.ipc.skip = True
+    step.refpix.odd_even_columns = True
```

Comment:
These five parameters (odd_even_columns, use_side_ref_pixels, side_smoothing_length, side_gain, and odd_even_rows) are not used for IRS2 data.

Reply:
OK, I'll remove them, since they're not needed.